### PR TITLE
completed catalog-hovers

### PR DESCRIPTION
**Pull request by andriy-shymkiv**
- [DEMO LINK](https://andriy-shymkiv.github.io/layout_catalog_hovers/)

### CODE REVIEW

File: src/styles/_header.scss [hunk 1]
```diff
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: transform 0.3s, color 0.3s;
```
Comment (Olexandr6) [inline src/styles/_header.scss:40]: `0.3s` values are used in several places, so it makes sense to create a variable for this value.
Let's do this!

**Review by G-YaSha — APPROVED** [review]
Almost done🔥

**Review by G-YaSha — CHANGES_REQUESTED** [review]
.

File: src/styles/_header.scss (hunk 1 above)
Comment (Andriy-Shymkiv) [reply src/styles/_header.scss:40]: okay

**Review by G-YaSha — APPROVED** [review]

**Pull request closed without merging**